templates/tab/js/default/.env.teamsfx.local

```diff
@@ -0,0 +1,13 @@
+# TeamsFx will overwrite the following variable values when running debug
+SSL_CRT_FILE=
+SSL_KEY_FILE=
+PORT=
+BROWSER=none
+HTTPS=true
```

Comment:
I don't think toolkit will overwrite these two variables, maybe move to below?
```
# Following variables can be customized or you can add your owns
# FOO=BAR
BROWSER=none
HTTPS=true
```

Reply:
Confirmed with @swatDong, TTK will overwrite the two variables:

- `BROWSER=none` for not opening browser automatically
- `HTTPS=true` since Teams app required HTTPS

Reply:
`BROWSER` does not matter, but `HTTPS` will be reset to *true* by toolkit. `HTTPS` is set together with the SSL* settings.

Reply:
@swatDong  I think `BROWSER=none` is also required? Otherwise, there will be two browsers opened. And I have tried current experience, both two variables will be overwritten.

Reply:
Ah yes you are right, to ensure F5 does not have confusing behavior.